src/Shared/Components/SelectPicker/FilterSelectPicker.tsx

```diff
@@ -39,6 +41,7 @@ const FilterSelectPicker = ({
     }
 
     const handleSelectOnChange: SelectPickerProps<number | string, true>['onChange'] = (selectedOptionsToUpdate) => {
+        setInputValue(inputValue)
```

Comment:
wouldn't this persist the searched value on closing the menu as well?

Reply:
No it does not persist